GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -275,6 +255,11 @@ public override bool TryDownloadNewestVersion(out string errorMessage)
                 return false;
             }
 
+            if (!this.EnsureNuGetFeedInitialized(out errorMessage))

```

Comment:
We are calling `EnsureNuGetFeedInitialized` above on line 188 as well.  Do we need to call this twice?

Reply:
With the `OrgNuGetUpgrader`, it is possible to call "TryDownloadNewestVersion" without calling the  "TryQueryNewestVersion" implementation in this method.

I have a refactoring change coming up that should make this a bit clearer.

Reply:
Oops I totally missed that we were in a different method here